waku/v2/node/discv5/waku_discv5.nim

```diff
@@ -77,7 +77,8 @@ proc findRandomPeers*(wakuDiscv5: WakuDiscoveryV5): Future[Result[seq[RemotePeer
   let discoveredNodes = await wakuDiscv5.protocol.queryRandom()
   
   ## Filter based on our needs
-  let filteredNodes = discoveredNodes.filter(isWakuNode) # Currently only a single predicate
+  # let filteredNodes = discoveredNodes.filter(isWakuNode) # Currently only a single predicate
+  let filteredNodes = discoveredNodes # we do not filter based on ENR in the first waku discv5 beta stage
```

Comment:
I'd rather not make the copy here. In other words, just use `discoveredNodes`, until some time in future we decide to filter again. For now `filteredNodes` can be removed.

Reply:
Done. I left a comment about filtering.